common/calibration-model.h

```diff
@@ -26,12 +28,19 @@ namespace rs2
 
     private:
         void draw_float4x4(std::string name, float3x3 & feild, const float3x3& original, bool& changed);
+        void draw_intrinsics( std::string name, mini_intrinsics & feild, const mini_intrinsics & original, bool & changed );
```

Comment:
Typo feild==>field
What is the purpose of this variable? Try giving it a more appropriate name like "current" or "output"